src/app/add-data-set-dialog/add-data-set-dialog.component.html

```diff
@@ -0,0 +1,30 @@
+<h2 mat-dialog-title>Add new projects</h2>
+<mat-dialog-content class="mat-typography">
+    <button mat-icon-button (click)="addEmptyProject()">
+        <mat-icon>add_circle</mat-icon>
+      </button>
+    <form class="example-form">
```

Comment:
It would be good to avoid generic class names like "example-...". This occurs in multiple places, so go through the code and change all the class names that look like this.